server/pkg/tfchain/client.go

```diff
@@ -642,3 +655,34 @@ func (c *Client) SwapToStellar(ctx context.Context, conState jsonrpc.State, args
 
 	return state.client.SwapToStellar(state.identity, args.TargetStellarAddress, *args.Amount)
 }
+
+func (c *Client) AwaitTransactionOnTfchainBridge(ctx context.Context, conState jsonrpc.State, memo string) error {
```

Comment:
`ctx` is an unused parameter, can we also include that if the context is canceled, this function exits?